teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, name: 'Unknown')
+    super(id, age, name)
```

Comment:
- [ ] You forgot to pass the `parent_permission` as a parameter in `initialize` and in super`, please kindly fix that 🙃 🙃